src/annotator/integrations/test/image-text-layer-test.js

```diff
@@ -150,37 +168,68 @@ describe('ImageTextLayer', () => {
       imageText
     );
 
-    assert.equal(textLayer.container.textContent, 'first line second line ');
-    const wordSpans = getWordBoxes(textLayer);
-    assert.equal(wordSpans.length, imageText.split(/\s+/).length);
+    assert.equal(textLayer.container.textContent, 'first line second line');
+    const wordBoxes = getWordBoxes(textLayer);
+    assert.equal(wordBoxes.length, imageText.split(/\s+/).length);
     assert.deepEqual(
-      wordSpans.map(ws => ws.textContent),
-      ['first', 'line', 'second', 'line']
+      wordBoxes.map(ws => ws.textContent),
+      ['first ', 'line ', 'second ', 'line']
     );
 
     const imageBox = image.getBoundingClientRect();
-    const wordBoxPositions = wordSpans.map(span => {
-      const wordBox = span.getBoundingClientRect();
+    const wordBoxPositions = wordBoxes.map(wordEl => {
+      const rect = untransformedBoundingRect(wordEl);
       return [
-        wordBox.left - imageBox.left,
-        wordBox.top - imageBox.top,
-        wordBox.width,
-        wordBox.height,
-      ].map(coord => Math.floor(coord));
+        rect.left - imageBox.left,
+        rect.top - imageBox.top,
+        rect.width,
+        rect.height,
+      ].map(coord => Math.round(coord));
     });
 
     const imageWidth = parseInt(image.style.width);
     const imageHeight = parseInt(image.style.height);
 
     const expectedPositions = [
-      expectedBoxOffsetAndSize(imageWidth, imageHeight, 0, 0, 'first'),
-      expectedBoxOffsetAndSize(imageWidth, imageHeight, 0, 6, 'line'),
-      expectedBoxOffsetAndSize(imageWidth, imageHeight, 1, 0, 'second'),
+      expectedBoxOffsetAndSize(imageWidth, imageHeight, 0, 0, 'first '),
+      expectedBoxOffsetAndSize(imageWidth, imageHeight, 0, 6, 'line '),
+      expectedBoxOffsetAndSize(imageWidth, imageHeight, 1, 0, 'second '),
       expectedBoxOffsetAndSize(imageWidth, imageHeight, 1, 7, 'line'),
     ];
     assert.deepEqual(wordBoxPositions, expectedPositions);
   });
 
+  it('creates lines and columns in the text layer', () => {
+    const { image } = createPageImage();
+    const textLayer = createTextLayer(
+      image,
+      [
+        // Create two sets of char boxes which overlap. This will cause each
+        // set to be assigned to a separate column.
+        ...createCharBoxes('first line\nsecond line\n'),
+        ...createCharBoxes('third line\nforth line'),
```

Comment:
OK, this is pedantic, but it's `fourth` even in American spelling ;)

Reply:
🤦‍♂️